packages/react-router-dev/config/config.ts

```diff
@@ -351,66 +377,31 @@ export async function resolveReactRouterConfig({
     ...mergeReactRouterConfig(...presets, reactRouterUserConfig),
   };
 
-  // Log warning for incompatible vite config flags
-  if (!ssr && serverBundles) {
-    console.warn(
-      colors.yellow(
-        colors.bold("⚠️  SPA Mode: ") +
```

Comment:
Looks like I missed this error. This needs to be in the Vite plugin.

Reply:
@pcattori We don't currently have a good way to model warnings within the config loader, but in this case I actually figured that this warning is overkill since it makes sense that serverBundles would be ignored if you're not in SSR mode.